zebra-state/src/service/finalized_state.rs

```diff
@@ -572,6 +591,15 @@ impl FinalizedState {
     pub fn path(&self) -> &Path {
         self.db.path()
     }
+
+    /// Returns the stored `ValueBalance` for the best chain at the finalized tip height.
+    pub fn current_value_pool(&self) -> ValueBalance<NonNegative> {
+        let value_pool_cf = self.db.cf_handle("tip_chain_value_pool").unwrap();
+        let tip_height = &self.finalized_tip_height().unwrap();
```

Comment:
Let's explain why this method is only called when there is at least one block in the state.
Can you change this unwrap to an expect?

Reply:
Now you've deleted the genesis initialization code, you might want to use `unwrap_or` to provide the initial value.

Reply:
The call to `finalized_tip_height()` was removed at https://github.com/ZcashFoundation/zebra/pull/2599/commits/26b40d913974e1f4a8f74b53a9330127edb84567 as we are not going to store height.